.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,8 +7,8 @@
   Ticketd PRs should be prefixed with the ticket id, e.g. `[FOO-123] some really great stuff`
 -->
 
-- Ticket: []
-- Feature flag: n/a
+-   Ticket: []
```

Comment:
Why is the added space necessary?

Reply:
We should enable Prettier and run it on our code so PRs don't add unrelated changes.